v2/tools/generator/internal/codegen/pipeline/add_cross_resource_references.go

```diff
@@ -21,7 +21,7 @@ import (
 // AddCrossResourceReferencesStageID is the unique identifier for this pipeline stage
 const AddCrossResourceReferencesStageID = "addCrossResourceReferences"
 
-var armIDDescriptionRegex = regexp.MustCompile("(?i)(.*/subscriptions/.*?/resourceGroups/.*|ARM ID|Resource ID)")
+var armIDDescriptionRegex = regexp.MustCompile("(?i)(.*/subscriptions/.*?/resourceGroups/.*|ARM ID|Resource ID|resourceId)")
```

Comment:
:heart:

Reply:
There might be another case that I should add to that regex, I saw some other missed references in the Redis resources...

Reply:
Yeah unfortunately it's hard to be 100% perfect here. If you've seen other cases/examples definitely feel free to add them